### DISCUSSION TITLE
docker pull theia

### DISCUSSION THREAD

**Discussion by kamaitaching** [eclipse-theia/theia#10504] [General]
How to solve the problem that docker needs to log in to pull the image.
![image](https://user-images.githubusercontent.com/88232327/144734963-543bc004-cab2-4b58-8c02-d1194d110043.png)

**Comment by msujew** [comment]
Hi @KamaitachiTools,

we don't publish new Docker Theia images anymore, and the old ones where probably automatically removed. See https://github.com/theia-ide/theia-apps/issues/496 for more info. There's also an issue [here](https://github.com/theia-ide/theia-apps/issues/503) about this topic. You are recommended to build your own images.
